File: Web/Nikita12112002_NikitaGupta_1923it1159_3/Getting_Started/Count_Digits.cpp
#include <iostream>
using namespace std;

int count_digits(int n)
{
    if(n<=9)
    return 1;
    
    return count_digits(n/10)+1;
    
}
int main(int argc, char **argv){
    int n;
    cin >> n;
    
    //write your code here
    
   int s= count_digits(n);
    cout<<s<<endl;
}

File: Android/divyaagarwal24_DivyaAgarwal_2024it1100_2/GettingStarted/countdigits.java
import java.util.*;
  
  public class countdigits{
  
  public static void main(String[] args) {
      Scanner scn=new Scanner(System.in);
    int n,r,count=0;
    n=scn.nextInt();
    while(n!=0)
    {
        r=n%10;
        n=n/10;
        count++;
    }
    System.out.println(count);
    // write your code here  
   }
  }